src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2121,9 +2130,16 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     }
     else
     {
-        info[jss::load_factor] =
-            static_cast<double> (app_.getFeeTrack ().getLoadFactor ()) /
-                app_.getFeeTrack ().getLoadBase ();
+        auto const loadBase = app_.getFeeTrack().getLoadBase();
+        assert(!escalationMetrics ||
+            loadBase == escalationMetrics->referenceFeeLevel);
+
+        info[jss::load_factor] = static_cast<double> (loadFactor) / loadBase;
```

Comment:
I first started to get nervous when I noticed that the calculation for `load_factor_fee_escalation` is different between `!human` and `human`.  Then I noticed that it's not just `load_factor_fee_escalation`, it's all of the entries.

I guess the good news here is that the human form of both `load_factor` and `load_factor_server` are using the same scaling factor.  That's probably a good sign.

The human form of `load_factor_fee_escalation`, however uses a different scaling factor (probably because it's a fee, not a load?).  What's reassuring here, I guess, is that both `load_factor_fee_escalation` and `load_factor_fee_queue ` use the same scaling factor.  And they're both fees.